src/quo2/components/selectors/selectors.cljs

```diff
@@ -0,0 +1,129 @@
+       (when value
+         [rn/view {:style {:height 20
+                           :width 20}}
+          [icons/icon :main-icons2/check
+           {:size 20
+            :color (colors/theme-colors
+                    (colors/alpha colors/neutral-100 (if disabled? 0.3 1))
+                    (colors/alpha colors/white (if disabled? 0.3 1)))}]])]])))
+
+(defn checkbox-view [props]
+  (let [{:keys [value onChange disabled? containerStyle]} (bean/bean props)]
+    (reagent/as-element
+     [rn/touchable-without-feedback
+      {:on-press (when (and onChange (not disabled?)) onChange)}
+      [rn/view
+       {:style        (merge
+                       (js->clj containerStyle)
+                       {:height (if value 20 21)
+                        :width (if value 20 21)
+                        :border-radius 6
+                        :border-width (if value 0 1)
+                        :background-color (if value
+                                            (get-color value disabled?)
+                                            (colors/theme-colors
+                                             colors/white
+                                             (colors/alpha colors/neutral-80 0.4)))
+                        :border-color (if value :none
+                                          (get-color value disabled?))})
+
+        :accessibility-label (str "checkbox-" (if value "on" "off"))
+        :accessibility-role  :checkbox}
+       (when value
+         [rn/view {:style {:height 20
+                           :width 20}}
+          [icons/icon :main-icons2/check
+           {:size 20
+            :color (colors/alpha colors/white (if disabled? 0.3 1))}]])]])))
+
+(defn radio-view [props]
+  (let [{:keys [value onChange disabled? containerStyle]} (bean/bean props)]
+    (reagent/as-element
+     [rn/touchable-without-feedback
+      {:on-press (when (and onChange (not disabled?)) onChange)}
+      [rn/view
+       {:style        (merge
+                       (js->clj containerStyle)
+                       {:height 20
+                        :width 20
+                        :border-radius 40
+                        :border-width 1
+                        :border-color (get-color value disabled?)
+                        :background-color (colors/theme-colors colors/white
+                                                               (colors/alpha colors/neutral-80 0.4))})
+        :accessibility-label (str "radio-" (if value "on" "off"))
+        :accessibility-role  :checkbox}
+
+       [rn/view {:style {:margin-left :auto
+                         :height 14
+                         :width 14
+                         :background-color (when value (get-color value disabled?))
+                         :border-radius 40
+                         :margin-right :auto
+                         :margin-top :auto
+                         :margin-bottom :auto}}]]])))
+
+(defn toggle-view [props]
```

Comment:
[Nitpick]

If you want to explicitly tell consumers of this namespace to only use the memoized versions, then you could make these `*-view` vars private.

Reply:
done 👍